src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -49,6 +47,47 @@
 ERR_EXPIRED = "You cannot resend an expired invitation without regenerating the token."
 ERR_RATE_LIMITED = "You are being rate limited for too many invitations."
 
+_team_roles_description = """
+Configures the team role of the member. The two roles are:
+- `contributor`: Can view and act on issues. Depending on organization settings, they can also add team members.
+- `admin`: Has full management access to their team's membership and projects.
+```json
+{
+    "teamRoles": [
+        {
+            "teamSlug": "ancient-gabelers",
+            "role": "admin"
+        },
+        {
+            "teamSlug": "powerful-abolitionist",
+            "role": "contributor"
+        }
+    ]
+}
+```
+"""
+
+# Required to explictly define roles w/ descriptions because OrganizationMemberSerializer
+# has the wrong descriptions, includes deprecated admin, and excludes billing
```

Comment:
isn't admin only deprecated for orgs with team roles? i thought you needed a specific plan or higher for team roles

Reply:
It's available for free/team plans. We prob don't want to encourage people to switch to it as we'll eventually phase out admins, so I think it's better to leave it off for now.